internal/reports/create.go

```diff
@@ -93,11 +99,25 @@ func BuildCreateHandOffReport(clusterData CreateHandOff) bytes.Buffer {
 	handOffData.WriteString(" see vault for credentials ")
 	//handOffData.WriteString(strings.Repeat("-", 70))
 
-	handOffData.WriteString("\n--- Metaphor ")
+	handOffData.WriteString("\n--- Metaphor JS")
 	handOffData.WriteString(strings.Repeat("-", 57))
-	handOffData.WriteString(fmt.Sprintf("\n Development: %s", clusterData.MetaphorDevUrl))
-	handOffData.WriteString(fmt.Sprintf("\n Staging: %s", clusterData.MetaphorStageUrl))
-	handOffData.WriteString(fmt.Sprintf("\n Production:  %s\n", clusterData.MetaphorProductionUrl))
+	handOffData.WriteString(fmt.Sprintf("\n Development: %s", clusterData.MetaphorJsDevUrl))
+	handOffData.WriteString(fmt.Sprintf("\n Staging: %s", clusterData.MetaphorJsStageUrl))
+	handOffData.WriteString(fmt.Sprintf("\n Production:  %s\n", clusterData.MetaphorJsProductionUrl))
+	handOffData.WriteString(strings.Repeat("-", 70))
+
+	handOffData.WriteString("\n--- Metaphor Go")
+	handOffData.WriteString(strings.Repeat("-", 57))
+	handOffData.WriteString(fmt.Sprintf("\n Development: %s", clusterData.MetaphorGoDevUrl))
+	handOffData.WriteString(fmt.Sprintf("\n Staging: %s", clusterData.MetaphorGoStageUrl))
+	handOffData.WriteString(fmt.Sprintf("\n Production:  %s\n", clusterData.MetaphorGoProductionUrl))
+	handOffData.WriteString(strings.Repeat("-", 70))
```

Comment:
this is out dated.. every section is a function now, as discussed earlier.

Reply:
> this is out dated.. every section is a function now, as discussed earlier.

it's the current state of development, there is no reference to be followed so far. we will update it later when the new strategy is available.